rpc/core/src/model/message.rs

```diff
@@ -7,6 +7,7 @@ use std::{
     fmt::{Display, Formatter},
     sync::Arc,
 };
+// use wasm_bindgen::prelude::*;
```

Comment:
commented code (same on lines 546, 548)

Reply:
cleaned up.  also removed some obsolete comments in system metrics